packages/react/src/components/Popup/Popup.tsx

```diff
@@ -441,7 +441,7 @@ export default class Popup extends AutoControlledComponent<PopupProps, PopupStat
     const popupContentAttributes =
       accessibility.focusTrap || accessibility.autoFocus ? {} : popupWrapperAttributes
 
-    const popupContent = Popup.Content.create(content, {
+    const popupContent = Popup.Content.create(content || {}, {
```

Comment:
If we will pass `false` or `null` to this slot we will render nothing and it will break `Popper` and `Ref` components that expects children. I want to consider refactoring of this place more deeply.

Reply:
Are you going to create a separate issue for that?

Reply:
#1630 👍